mappingengine/start.rst

```diff
@@ -3,16 +3,24 @@
 ===============
 Getting Started
 ===============
-The FHIR Mapper (a .Net FHIR Mapping Engine) is available as a paid addon to the Vonk FHIR server. `Get in touch <https://fire.ly/contact/>`_ with us if you're interested in obtaining it for your needs.
+
+The FHIR Mapper is available as a paid plugin to the Vonk FHIR server. `Get in touch <https://fire.ly/contact/>`_ with us if you're interested in obtaining it for your needs. ::
+
+
+    Documentation is currently under construction! Please be patient.
+    
 
 Configuration
 -------------
 
-The section :ref:`configure_mapping_engine` explains how you can configure the mapping engine in your own Vonk server.
+The section :ref:`configure_mapping_engine` explains how you can configure the plugin in your own Vonk server.
 
 .. _mappingengine_run:
 
 Running transformations
 -----------------------
 
-Once you have setup the .Net FHIR Mapping Engine, you can start :ref:`setup_transforms`.
\ No newline at end of file
+
+Once you have setup the FHIR Mapper, you can start :ref:`setup_transforms`.
+
+Alternatively, you can try the FHIR Mapper in our public test sandbox: https://vonk.fire.ly/R3. The rest of the documentation will be using the public sandbox as a point of reference.
```

Comment:
Same here. I've edited this content while you changed the mapping engine to plugin.

Reply:
What do I need to change here? Why did you delete the part of the public endpoint?

Reply:
Because I checked the public endpoint and the plugin wasn't there, not in `/metadata` at least :O

Reply:
Not yet, will be next week

Reply:
Let's put it in there when it's live, better than having a mistake